cmd/pinniped/cmd/flag_types_test.go

```diff
@@ -52,7 +51,7 @@ func TestConciergeModeFlag(t *testing.T) {
 func TestCABundleFlag(t *testing.T) {
 	testCA, err := certauthority.New("Test CA", 1*time.Hour)
 	require.NoError(t, err)
-	tmpdir := testutil.TempDir(t)
+	tmpdir := t.TempDir()
```

Comment:
This must simply be an API change from the bump.

Reply:
This is a Pinniped-only refactor, it used to implement `TempDir` differently prior to `go1.14`